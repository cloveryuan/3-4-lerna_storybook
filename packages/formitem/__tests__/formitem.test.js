import MyyFormitem from "../src/formitem.vue"
import {mount}  from "@vue/test-utils"

describe('myy-formitem',()=>{
  test('formitem-prop',()=>{
    const wrapper = mount(MyyFormitem)
    expect(wrapper.vm.$el).toMatchSnapshot() // 生成快照
  })
})