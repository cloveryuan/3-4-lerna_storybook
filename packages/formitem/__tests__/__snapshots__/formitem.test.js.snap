// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`myy-formitem formitem-prop 1`] = `
<div>
  <label>
    
  </label>
   
  <div>
     
    <!---->
  </div>
</div>
`;
